tests/test_animation.py

```diff
@@ -72,7 +75,9 @@ def test_send_all_args(self, bot, chat_id, animation_file, animation, thumb_file
 
         assert isinstance(message.animation, Animation)
         assert isinstance(message.animation.file_id, str)
+        assert isinstance(message.animation.file_unique_id, str)
         assert message.animation.file_id != ''
+        assert message.animation.file_unique_id != ''
```

Comment:
shouldn;t we actually test it is the expected unique_id? (also for the others I suppose

Reply:
Yes, that would probably be better. I thinks this was just copy-pasted from the exsting tests for `file_id`. I'll try and adjust everywhere

Reply:
wait, arent we creating a new file here?

Reply:
we would need to send an animation via file_id to test if its the same unique file id no?

Reply:
Oh you're right. Then testing for `file_(unique_)id == self.file_(unique_)id` doesn't make sense.